RobotMbed/src/main/java/de/fhg/iais/roberta/visitor/codegen/MicrobitPythonVisitor.java

```diff
@@ -364,10 +364,10 @@ public Void visitPinWriteValueAction(PinWriteValueAction<Void> pinWriteValueActi
     }
 
     @Override
-    public Void visitPinSetPullAction(PinSetPullAction<Void> pinSetPullAction) {
```

Comment:
do you have a microbit to check whether the runtime is updated?

Reply:
no i don't have a microbit but the information i received is that the runtime is not updated